content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -935,6 +1014,64 @@ const counter = myMeter.createCounter('my.counter');
 cntr.add(1, { 'some.optional.attribute': 'some value' });
 ```
 
+### Configure Metric Views
+
+A Metric View provides developers with the ability to customize metrics exposed by the Metrics SDK.
+
+#### Selectors
+
+To instantiate a view, one must first select a target instrument. The following are valid selectors for metrics:
+- `instrumentType`
+- `instrumentName`
+- `meterName`
+- `meterVersion`
+- `meterSchemaUrl`
+
+Selecting by `instrumentName` (of type string) has support for wildcards, so you can select all instruments using `*` or select all instruments whose name starts with `http` by using `http*`.
+
+#### Examples
+
+Filter attributes on all metric types:
+
+```js
+new View({
+  // only export the attribute 'environment'
+  attributeKeys: ['environment'],
+  // apply the view to all instruments
+  instrumentName: '*',
+})
+```
+
+Drop all instruments with the meter name `pubsub`:
+
+```js
+const dropView = new View({
+  aggregation: new DropAggregation(),
+  meterName: 'pubsub',
+});
+```
+
+Define explicit bucket sizes for the Histogram named `http.server.duration`:
+
+```js
+const histogramView = new View({
+  aggregation: new ExplicitBucketHistogramAggregation([0, 1, 5, 10, 15, 20, 25, 30]),
+  instrumentName: 'http.server.duration',
+  instrumentType: InstrumentType.HISTOGRAM,
+});
+```
+
+#### Attach to meter provider
+
+Once a view has been configured, attach it to the corresponding meter provider:
+```js
+const meterProvider = new MeterProvider({
+  views: [
+    histogramView
+  ]
+});
```

Comment:
Optional: I think in most cases, users will have multiple views. We may be able to tie it together with the previous section and show all of the views applied to the `MeterProvider` like so  :slightly_smiling_face::

```suggestion
Once views have been configured, attach them to the corresponding meter provider:
```js
const meterProvider = new MeterProvider({
  views: [
    limitAttributesView,
    dropView,
    histogramView
  ]
});
```